service/internal/db/db_test.go

```diff
@@ -0,0 +1,57 @@
+package db
+
+import (
+	"reflect"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewSQLClientWithOptions(t *testing.T) {
+	tests := []struct {
+		name      string
+		dbProps   map[string]string
+		dbOptions map[string]string
+	}{
+		{
+			name: "sqlClientWithOptions",
+			dbProps: map[string]string{
+				"host":     "localhost",
+				"database": "argocloudops",
+				"user":     "argocd",
+				"password": "argocd_password",
+				"options":  "sslrootcert=rds-ca.pem sslmode=verify-full",
+			},
+			dbOptions: map[string]string{
+				"sslrootcert": "rds-ca.pem",
+				"sslmode":     "verify-full",
+			},
+		},
+		{
+			name: "sqlClientWithoutOptions",
+			dbProps: map[string]string{
+				"host":     "localhost",
+				"database": "argocloudops",
+				"user":     "argocd",
+				"password": "argocd_password",
+				"options":  "",
+			},
+			dbOptions: map[string]string{},
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			db, err := NewSQLClient(
+				tt.dbProps["host"],
+				tt.dbProps["database"],
+				tt.dbProps["user"],
+				tt.dbProps["password"],
+				tt.dbProps["options"],
+			)
+
+			assert.NoError(t, err)
+			assert.True(t, reflect.DeepEqual(db.options, tt.dbOptions))
```

Comment:
Asserting `true` will not provide info on what the issue/diff is, instead we can just use `Equal`:
```suggestion
			assert.Equal(t, db.options, tt.dbOptions)
```

Which will look like this upon failure:

```
--- FAIL: TestNewSQLClientWithOptions (0.00s)
    --- FAIL: TestNewSQLClientWithOptions/sqlClientWithOptions (0.00s)
        db_test.go:55:
                Error Trace:    /Users/tbishop/dev/github.com/cello-proj/cello/service/internal/db/db_test.go:55
                Error:          Not equal:
                                expected: map[string]string{"sslmode":"verify-full", "sslrootcert":""}
                                actual  : map[string]string{"sslmode":"verify-full", "sslrootcert":"rds-ca.pem"}

                                Diff:
                                --- Expected
                                +++ Actual
                                @@ -2,3 +2,3 @@
                                  (string) (len=7) "sslmode": (string) (len=11) "verify-full",
                                - (string) (len=11) "sslrootcert": (string) ""
                                + (string) (len=11) "sslrootcert": (string) (len=10) "rds-ca.pem"
                                 }
                Test:           TestNewSQLClientWithOptions/sqlClientWithOptions
```